force-app/main/default/triggers/OpportunityTrigger.trigger

```diff
@@ -0,0 +1,104 @@
+/*Opportunity trigger should do the following:
+* 1. Validate that the amount is greater than 5000.
+* 2. Prevent the deletion of a closed won opportunity for a banking account.
+* 3. Set the primary contact on the opportunity to the contact with the title of CEO.
+*/
+
+
+
+/*
+    * Question 5
+    * Opportunity Trigger
+    * When an opportunity is updated validate that the amount is greater than 5000.
+    * Error Message: 'Opportunity amount must be greater than 5000'
+    * Trigger should only fire on update.
+    */
+
+trigger OpportunityTrigger on Opportunity (before insert, before update, before delete) {
+
+if (trigger.isUpdate) {
+    for (Opportunity oppy : Trigger.new) {
+        if (oppy.Amount < 5000) {
```

Comment:
```suggestion
        if (oppy.Amount <= 5000) {
```
The requirements say GREATER than 5000, so I would assume 5000 should also error.